src/message.c

```diff
@@ -92,6 +92,8 @@ static bool request_handler(const struct sip_msg *msg, void *arg)
 
 	ua = uag_find_msg(msg);
 	if (!ua) {
+		info("ua: %r: UA not found: %H\n",
+		     &msg->from.auri, uri_encode, &msg->uri);
```

Comment:
Perhaps it is better if the log prefix matches the name of the .c file, i.e. `message: xxx` ?